### ISSUE TITLE
treatment histories

### ISSUE THREAD

**Issue by harryhoch** [phenopackets/phenopacket-format#2]
" For example, it would not record any history of treatment". 

Certain treatments may be important for discussions of medication sensitivities, etc.

**Comment by cmungall** [comment]
No question there, also history of stressors and exposures, lab results, images, EKGs, etc. These can all be of course thrown into phenotype term bins but my understanding is that the precise measurements and observations would be out of scope here, as not being something that can be practically submitted to registries and case reports etc

**Comment by harryhoch** [comment]
Agreed. Another good point to make explicit in reqs. Would it be safe here to say that we're looking at interpretations of meds, labs, etc. that influence understanding of patient condition, as opposed to raw values/data?
